### DISCUSSION TITLE
Exercise 15: Show that Paxos is not live.

### DISCUSSION THREAD

**Discussion by dag7dev** [sapienzastudentsnetwork/distributed-systems#40] [Exercises]
**Exercise 15:** Show that Paxos is not live.

**Comment by susannacifani** [comment]
![image](https://github.com/sapienzastudentsnetwork/distributed-system-2023-2024/assets/73530772/266df706-5f32-4e89-993d-ea8de3a1c27d)

**Comment by cornflexxx** [comment]
Consider a proposer $p$ which sends a $\langle \text{PREPARE }i\rangle$ and the acceptor sends the corresponding $\langle \text{PREPARE} \rangle$ message. Then $p$ sends the accept message, but before receiving the accept message the acceptor  receive another  $\langle \text{PREPARE }i'\rangle$, with $i'>i$  from another proposer $p'$ preventing $i$ to conclude successfully. Furthermore, round $i'$ can be interrupted in a similar manner and can continue in this way indefinitely ($\implies$ Paxos not live).
